### PR TITLE
NFS export options for Filestore

### PR DESCRIPTION
**Pull request by wiktorn**
Add NFS export options for Filestore that are necessary for clusters running in non-RFC1918 network ranges.

### CODE REVIEW

File: modules/file-system/filestore/variables.tf [hunk 1]
```diff
@@ -114,6 +114,16 @@ variable "connect_mode" {
   }
 }
 
+variable "nfs_export_options" {
+  description = "Define NFS export options."
+  type = object({
```
Comment (mr0re1) [inline modules/file-system/filestore/variables.tf:119]: `nfs_export_options` block can be repeated multiple times (up to 10), let's make it `list` of objects instead.

Comment (wiktorn) [reply modules/file-system/filestore/variables.tf]: Made it a non-nullable list.

**Review by mr0re1 — APPROVED** [review]

**Comment by mr0re1** [comment]
Thank you for contribution!

**Pull request merged**